src/utils/createReadStreamInternal.ts

```diff
@@ -363,6 +363,9 @@ export function createReadStreamInternal(
     rowStream = pumpify.obj([requestStream, chunkTransformer, toRowStream]);
 
     metricsCollector.wrapRequest(requestStream);
+    requestStream.on('data', () => {
+      metricsCollector.onResponse();
+    });
```

Comment:
Doesn't this conflict with https://github.com/googleapis/nodejs-bigtable/pull/1658?